i18n/en.json

```diff
@@ -2209,7 +2209,7 @@
   },
   {
     "id": "api.templates.signin_change_email.subject",
-    "translation": "[{{ .SiteName }}] You updated your sign-in method on {{ .SiteName }}"
+    "translation": "[{{ .SiteName }}] Your sign-in method has been updated"
```

Comment:
@wget Let me know if you have any questions about this one :) 

I think we just need to update the relevant unit test to expect the new string.